tests/roxctl/bats-tests/local/roxctl-netpol-connectivity-map-development.bats

```diff
@@ -605,6 +565,253 @@ hello-world/workload-a[Deployment] is not protected on Egress'
   assert_output "$normalized_expected_output"
 }
 
+@test "roxctl-development netpol connectivity map generates connlist for input resources with admin network policies" {
```

Comment:
I think it should be sufficient to just add the `txt` format for the added ANP/BANP tests  (both here and in diff).
There are other tests focusing on these other formats.

Reply:
done